repo/config/init.go

```diff
@@ -85,6 +85,7 @@ func datastoreConfig() (Datastore, error) {
 		StorageGCWatermark: 90, // 90%
 		GCPeriod:           "1h",
 		HashOnRead:         false,
+		BloomFilterSize:    0,
```

Comment:
Yes it is, we decided to do that until we get better metrics, repo size caching (important information to size to bloom filter correctly) and bloom filter expansion.